komodo-modeshape-vdb/src/test/resources/config/komodo.cnd

```diff
@@ -0,0 +1,408 @@
+/*
```

Comment:
The repository config file used in tests for this project reference `config/komodo.cnd` (which previously did not exist in the project). I was getting errors running tests unless I added the file to this project. Do the tests run for you without this file?